src/installation_guide/text.adoc

```diff
@@ -410,5 +410,3 @@ docker-compose up
 Now you can head back to follow the section titled <<Creating a User and Running a Command>>,
 as the rest isn't Slack specific. The only caveat is that when creating a
 chat-handle, you'll need to specify `--chat-provider=hipchat` instead.
-
-
```

Comment:
Oh so this is dumb, but I think each chapter needs these empty lines at the end otherwise we get those weird table of contents issues.